website/docs/addon_traypublisher_artist.md

```diff
@@ -0,0 +1,106 @@
+---
+id: addon_traypublisher_artist
+title: Tray Publisher
+sidebar_label: Tray Publisher
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.TrayPublisher_Badge}
+</ReactMarkdown>
+
+Tray Publisher is a standalone tool for publishing products into AYON pipeline.
+It can be accessed via UI or command line.
+
+
+## Tray Publisher
+How to use it ? 
+1. Access the tool via Tray menu.
+![](assets/traypublisher/artist/open_publisher.gif)
+2. Create and Publish 
+   a. Select AYON Folder path
+   b. Select a Task
+   c. Select a product Type
+   d. Drop your files
+   e. Click create
+   f. Finally, click publish.
+![](assets/traypublisher/artist/publish_a_product.gif)
+3. Then you'll be able to access the published product via AYON Loader.
+![](assets/traypublisher/artist/check_in_loader.gif)
+
+## Tray Publisher CLI Interface
+
+:::info Tray CLI Interface
+Tray Publisher provides a CLI Interface (Windows Only).
```

Comment:
Why windows only?

Reply:
Is the CLI accessible on Linux and Mac ? 
I think I misinterpreted some info as the examples are on windows only. 
e.g. 
```python 
cd "C:\Program Files\Ynput\[AYON Launcher Version]"
.\ayon_console.exe addon traypublisher launch
```

Reply:
I've removed that note here
https://github.com/ynput/ayon-documentation/pull/164/commits/94ee4ce502a618c76943f659346db2f433dafd84